tests/js/tests.js

```diff
@@ -27,12 +27,38 @@ test.describe("top nav bar", () => {
     await expect(pageToC).toHaveScreenshot();
```

Comment:
Thanks for adding all the dark mode tests! I'm a little concerned about duplication because it makes the tests more brittle and harder to update when we make changes (I've several times had to tweak the tests)

Here's an alternative approach you could use to DRY (don't repeat yourself):

```typescript
for (const colorScheme of ["light", "dark"] ) {
  test.describe(`{colorScheme}` mode`, () => {
     test.beforeEach(async ({ page }) => {
         if (colorScheme === "dark") {
             await toggleTheme(page);
         }
     })

     test.describe("top nav bar", () => { ... }))
  })
}
```

I know you have two tests that toggle through every mode. Those would live outside of the for loop and be top-level `test.describe()` calls like they currently are

Reply:
Done

Reply:
Hold on I'm going to refactor the top nav bar tests too

Reply:
Done